decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -265,7 +265,20 @@
         visibility: "all"
       ) do
         proposal = Decidim::Proposals::Proposal.new(params)
-        proposal.add_coauthor(participatory_space.organization)
+        
```

Comment:
```suggestion
```
At least from the diff page, I couldn't see the error... From the "[View page](https://github.com/decidim/decidim/blob/5af63ae4cb1da52f8cb587b8476d12c120769ee7/decidim-proposals/lib/decidim/proposals/component.rb#L268)" link, it's much clearer -- there are some spaces in the empty line that we need to get rid of